vendor/vendor.json

```diff
@@ -47,25 +41,7 @@
 			"path": "github.com/smartystreets/goconvey/convey/reporting",
 			"revision": "a2d5d788f3afbce0989463251ccc5492fad832df",
 			"revisionTime": "2015-12-28T14:47:19-07:00"
-		},
-		{
-			"checksumSHA1": "6U7dCaxxIMjf5V02iWgyAwppczw=",
-			"path": "golang.org/x/crypto/ssh/terminal",
-			"revision": "8c653846df49742c4c85ec37e5d9f8d3ba657895",
-			"revisionTime": "2018-02-19T16:33:59Z"
-		},
-		{
-			"checksumSHA1": "+UMEPHX8wvMkCxa+ofAmBKlqdYg=",
-			"path": "golang.org/x/sys/unix",
-			"revision": "f6cff0780e542efa0c8e864dc8fa522808f6a598",
-			"revisionTime": "2018-02-20T20:26:15Z"
-		},
-		{
-			"checksumSHA1": "eQq+ZoTWPjyizS9XalhZwfGjQao=",
-			"path": "golang.org/x/sys/windows",
-			"revision": "f6cff0780e542efa0c8e864dc8fa522808f6a598",
-			"revisionTime": "2018-02-20T20:26:15Z"
 		}
 	],
-	"rootPath": "github.com/grindlemire/death"
```

Comment:
I think this was a mistake from your last merge?

Reply:
Yea that was my bad. 👍